app/Db/pu610k.db

```diff
@@ -48,6 +48,28 @@ record(bo, "$(BASE):$(CHAN):ENABLE_RBV") {
     field(ONAM, "Enabled")
 }
 
+record(calc, "$(BASE):CHILLER_OK") {
+    field(DESC, "LPL Glass Rod chillers on")
+    field(INPA, "$(CHILL):1")
+    field(INPB, "$(CHILL):2")
+    field(INPC, "$(CHILL):3")
+    field(PINI, "YES")
+    field(CALC, "A&&B&&C") 
+    field(SCAN, "1 second")
+    field(PROC, "$(BASE):CHILLER_INTERLOCK.PROC")
+}
+
+record(calcout, "$(BASE):CHILLER_INTERLOCK") {
```

Comment:
I'm not sure why this record is needed. You're not modifying the value. Seems like the `CHILLER_OK` record could be a `calcout` instead of `calc`. The `MODE_ALLOW` PV seems to handle both potential cases for the `CHILLER_OK` PV.

Maybe I'm missing something here.

Reply:
Looking at it more, seems right, we can make this one record.

Reply:
would like to make some fault indication here somewhere as well.